.gitignore

```diff
@@ -1,2 +1,5 @@
 composer.lock
 vendor/
+
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio
+.idea/
```

Comment:
IMO, this does not belong in the project ignore rules. You should ignore it in your dev environment instead (either globally or per project): https://help.github.com/articles/ignoring-files/#create-a-global-gitignore